### DISCUSSION TITLE
ForceGraph3D node module dependency issue. . /webgpu

### DISCUSSION THREAD

**Discussion by jbudus** [vasturiano/react-force-graph#547] [Q&A]
I recently ran an npm update for my nextjs/react app. (next 14 not 15) 

I now get this compiling error but connot figure out why.

./node_modules/three-render-objects/dist/three-render-objects.mjs:2:1

Package path ./webgpu is not exported from package /node_modules/three.

My package.json contains this:

"react-force-graph-3d": "^1.24.5",
Any help would be very much appreciated as I've spent hours on this issue.

**Comment by vasturiano** [comment]
@jbudus please update the version of `three` in your dependency tree to at least `0.168`. Earlier versions don't have the `webgpu` export, thus leading to this error.
